Bencodex/Types/List.cs

```diff
@@ -19,6 +20,8 @@ public List(IEnumerable<IValue> value)
             _value = value?.ToImmutableArray() ?? ImmutableArray<IValue>.Empty;
         }
 
+        public static List Empty => default(List);
```

Comment:
Shouldn't this be recorded to the changelog?

Reply:
I also recorded it at 56c665b. Also, there was no record for `Bencodex.Types.Dictionary.Empty`, also recorded it at ecb061b.